### PR TITLE
3078 otp expiration timer

### PR DESCRIPTION
**Pull request by cakekoa**
# What does this PR do?

Fixes part of #3078 by displaying a countdown timer for the OTP expiration.

- Displays a countdown timer for command expiration

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by mssalvatore — COMMENTED** [review]
A video clip would help speed review along.

**Review by ilija-lazoroski — CHANGES_REQUESTED** [review]
I would expect the command to be refreshed automatically if it is expired. Maybe that is not something we want but as a user experience, I wouldn't want to click Refresh after the timer expires.

https://user-images.githubusercontent.com/15820737/230332346-87d402f5-0cf2-4038-9c2d-9baa7ab45816.mov

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export function CommandExpirationTimer({ minutes, seconds }) {
+
+    function formatDigits(number) {
+      return number.toLocaleString('en-US', { minimumIntegerDigits: 2, useGrouping: false})
+    }
+
+    function generateText() {
+      if (minutes === 0 && seconds === 0) {
+        return 'Command expired';
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx:11]: Can we regenerate the command and restart the timer here?

**Comment by mssalvatore** [comment]
> I would expect the command to be refreshed automatically if it is expired.

+1

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx (hunk 1 above)
Comment (cakekoa) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx:11]: Note: The `useTimer` method allows one to register an `onExpire` callback

Comment (cakekoa) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx:11]: https://user-images.githubusercontent.com/11077625/230366820-af7088c3-5143-4ab2-894b-ff6e5d362be9.mov

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**